### DISCUSSION TITLE
Splitting ape + cue into individual tracks

### DISCUSSION THREAD

**Discussion by ie377** [enzo1982/freac#490] [Audio Format Discussion]
Hi! I have a bunch of cds ripped to the cursed ape+cue format. How do I convert them to MP3 with Fre:AC (MX Linux), not a bulk file but individual tracks?

I read I need some mac codec for that, but I can't find it. Thank you in advance

**Comment by enzo1982** [comment]
If you're using the Flatpak, Snap or AppImage build of fre:ac, you won't need an extra codec. Monkey's Audio is already included with those.

It should be sufficient to open the .cue file in fre:ac. The individual tracks should then show up in the joblist and you can convert them to individual MP3 files.

**Reply by ie377** [reply]
Thank you! 
My Freac couldn't open a cue file as it was written by a windows pc in windows-1251 encoding, that's why the program could not open files mentioned in the .cue. Files names were damaged in .cue. I changed encoding of .cue to utf-8 and Freac finally saw the files.

**Reply by ie377** [reply]
Also Freac is a superb app, the best one out there! Super reliable, versatile, extracts audio from video files easily, uses all cores, stitches files together - it's a masterpiece of a software!

**Reply by enzo1982** [reply]
Thank you! Always great to read that!
